### PR TITLE
Update and tweak the "getting started" instructions

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: src/dev/getting_started.md [hunk 1]
```diff
@@ -2,26 +2,23 @@
 
 This guide describes how to get up and running with yk.
 
-## Repos
+## Repositories
 
-yk is spread over a handful of repos, but the two you are likely to need to clone are:
+yk is spread over several git repositories, but you will need to downloads and
```
Comment (vext01) [inline src/dev/getting_started.md:7]: s/downloads/download/

**Comment by vext01** [comment]
Just a typo.

File: src/dev/getting_started.md (hunk 1 above)
Comment (ltratt) [reply src/dev/getting_started.md]: Fixed in 1b24c9f.

Comment (ltratt) [reply src/dev/getting_started.md]: Fixed in 1b24c9f.

**Comment by vext01** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**